client/webserver/site/src/html/bodybuilder.tmpl

```diff
@@ -104,7 +104,7 @@
 {{end}}
 
 {{define "bottom"}}
-<script src="/js/entry.js?v=88986a77|0cdffb94"></script>
+<script src="/js/entry.js?v=ebcfcdea|a99362bc"></script>
```

Comment:
Getting the same values locally.

@ukane-philemon Do you have `core.autocrlf` set to something with git?

Reply:
> @ukane-philemon Do you have core.autocrlf set to something with git?

No, I don't.